app/src/components/nav/AppShell.tsx

```diff
@@ -87,7 +88,7 @@ const NavSidebar = () => {
               </Text>
               <IconLink icon={BsGearFill} label="Project Settings" href="/settings" />
               <IconLink icon={IoSpeedometerOutline} label="Usage" href="/usage" />
-              {/* <IconLink icon={LiaFileInvoiceDollarSolid} label="Billing" href="/billing" /> */}
+              <IconLink icon={LiaFileInvoiceDollarSolid} label="Billing" href="/billing" />
```

Comment:
I'm excited to get this in! Do you think we could hold off on showing the icon link until we've validated that everything looks good in production? I'd like to look these changes over in production before we roll it out.

Reply:
Sure, I have removed the billing link.